.github/workflows/cla.yml

```diff
@@ -0,0 +1,55 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened,closed,synchronize]
+
+permissions:
+  actions: write
+  contents: write 
+  pull-requests: write
+  statuses: write
+
+jobs:
+  CLAAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Harden the runner (Audit all outbound calls)
+        uses: step-security/harden-runner@4d991eb9b905ef189e4c376166672c3f2f230481 # v2.11.0
+        with:
+          egress-policy: audit
+          
+      - name: Checkout Private Repo for Allowlist
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 #v4.2.2
```

Comment:
Do we prefer to pin the commit? Imagine if someone leaves the ORG. You will update the list but then we will need to update the commit here.

Reply:
This is specifically for the checkout action commit...for private repo checkout it always does latest commit and i have few admins added to that repo which will manage to keep it up to date.

Reply:
Great